RNSnackbar.podspec

```diff
@@ -0,0 +1,20 @@
+require 'json'
+
+package = JSON.parse(File.read('package.json'))
+
+Pod::Spec.new do |s|
+  s.name                = 'RNSnackbar'
+  s.version             = package['version']
+  s.summary             = package['description']
+  s.description         = package['description']
+  s.homepage            = package['homepage']
+  s.license             = package['license']
+  s.author              = package['author']
+  s.source              = { :git => 'https://github.com/cooperka/react-native-snackbar.git' }
```

Comment:
This podspec will help you linking your library same as 'react-native link react-native-snackbar'. The package version will be pulled from the project's package.json. Nothing to do with the tags because you do not pull directly from git, but add it locally from your node_modules folder.
To use this podspec, simply put this in your Podfile, then you go with 'pod install' instead of react-native link
    pod 'RNSnackbar', :path =>'../node_modules/react-native-snackbar'